src/envo/envo-edit.owl

```diff
@@ -26,7 +26,7 @@ Import(<http://purl.obolibrary.org/obo/envo/imports/uberon_import.owl>)
 Import(<http://purl.obolibrary.org/obo/envo/modules/entity_attribute_location.owl>)
 Import(<http://purl.obolibrary.org/obo/envo/imports/chebi_import.owl>)
 Import(<http://purl.obolibrary.org/obo/envo/imports/iao_import.owl>)
-Annotation(<http://purl.org/dc/elements/1.1/creator> "http://orcid.org/0000-0002-4366-3088")
+Annotation(<http://purl.org/dc/elements/1.1/creator> <http://orcid.org/0000-0002-4366-3088>)
```

Comment:
While fixing the nanocrediting APs in this PR, I've made a bunch of substitutions of old ORCIDs rendered as strings with IRIs. More to be done here in other PRs.